app/src/main/res/xml/searchable.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+    <?xml version="1.0" encoding="utf-8"?>
```

Comment:
hey bro, this line of code is causing errors

Reply:
I think this line of code should be in this file just once. You have it multiple times in this file. (** DUPLICATE CODE**)